src/main/main-controller.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+    'use strict';
+
+    class MainCtrl {
+      constructor($timeout, currentWindowService) {
+          this.$timeout = $timeout;
+          this.currentWindowService = currentWindowService;
+          currentWindowService.ready(this.onReady.bind(this));
+      }
+      onReady() {
+          this.window = this.currentWindowService.getCurrentWindow();
+      }
+
+      isSummarised() {
```

Comment:
I know we've been using the term "summary view" a lot but I think I'd prefer a name more like `isCompact`, and "compactView".